contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -82,6 +82,16 @@ contract DualAggregatorHarness is DualAggregator {
       recordedTimestamp: _recordedTimestamp
     });
   }
+
+  // helper function to inject 6 transmissions
+  function injectTransmissions() public {
```

Comment:
in order to facilitate more test cases in the future this should probably take in an array of each value (answer, observationsTimestamp, recordedTimestamp) and then iterate over them and call `setTransmission`.

another option would be to use the `ReportGenerator` contract from hendrik's PR and emit the full transmission itself to ensure e2e functionality works as expected.

Reply:
Good idea, I simplified the code as you requested in the last commit.